src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,428 @@
+        sub-01_sample-01_stain-03_<modality_suffix>.<ext>
+        sub-01_sample-01_stain-03_<modality_suffix>.json
+```
+In this example, the entity stain is used to distinguish images with different
+stains in separate files from the same sample.
+In the case where a single file contains different staining in each channel, the
+`stain-<label>` is omitted.
+
+Stains or antibodies SHOULD be indicated as appropriate in the `"SampleStaining"`,
+`"SamplePrimaryAntibodies"` and/or `"SampleSecondaryAntobodies"` keys in the sidecar JSON file,
+although the label may be different.
+
+If more than one run of the same sample, acquisition and stain are acquired during the same
+session, the [`run-<index>`](../99-appendices/09-entities.md#run) entity MUST be used:
+`_run-1`, `_run-2`, `_run-3`, and so on.
+If only one run was acquired the `run-<index>` can be omitted.
+
+The [`chunk-<index>`](../99-appendices/09-entities.md#chunk) entity is used when multiples
+regions (2D images or 3D volumes files) of the same physical sample are imaged with different
+fields of view, regardless if they overlap or not.
+
+In some cases, the chunks can be "ordered" and correspond to the displacement of the
+microscope stage, for example.
+In other cases, the chunks can be different images of the same sample with no explicit
+spatial relation between them.
+
+Examples of different chunks configurations can be seen in Figure 1.
+![Figure 1](images/microscopy_chunks1.png "Examples of Microscopy chunks")
+
+Figure 1: Examples of chunks configurations.
+
+-   a) ordered 2D chunks without overlap,
+-   b) ordered 2D chunks with overlap,
+-   c) unordered 2D chunks with and without overlap,
+-   d) and e) ordered 2D chunks on different 3D planes,
+-   f) ordered 3D chunks.
+
+<!--- The following example will be generated automatically with macros after community review. -->
+For example, as illustrated in Figure 1b:
+Six chunks (`chunk-01` to `chunk-06`) from the same brain sample (`sample-01`) of subject `sub-01`,
+are named:
+```Text
+sub-01/
+    microscopy/
+        sub-01_sample-01_chunk-01_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-01_<modality_suffix>.json
+        sub-01_sample-01_chunk-02_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-02_<modality_suffix>.json
+        sub-01_sample-01_chunk-03_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-03_<modality_suffix>.json
+        sub-01_sample-01_chunk-04_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-04_<modality_suffix>.json
+        sub-01_sample-01_chunk-05_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-05_<modality_suffix>.json
+        sub-01_sample-01_chunk-06_<modality_suffix>.<ext>
+        sub-01_sample-01_chunk-06_<modality_suffix>.json
+```
+The index number can be assigned arbitrarily and, in the case of "ordered" chunks, the chunks'
+positions relative to one another SHOULD be defined by an affine transformation matrix in the JSON
```

Comment:
```suggestion
positions relative to the previous one SHOULD be defined by an affine transformation matrix in the JSON
```
? since not sure what specific one is "one another"

Reply:
There is no "previous one" as the chunks' numbers are arbitrary. The position of each chunk is defined in a matrix in [chunk transformations](http://127.0.0.1:8000/04-modality-specific-files/10-microscopy.html#chunk-transformations).

Reply:
```suggestion
relative positions (in terms of scaling and traslation) SHOULD be defined by an affine transformation matrix in the JSON
```

Reply:
I integrated @guionar suggestion for chunks' relative positions, thank you!
@yarikoptic please let us know if it sounds good to you.